packages/manager/modules/pci/src/projects/projects.service.js

```diff
@@ -63,4 +64,9 @@ export default class {
       })
       .$promise.then((quotas) => map(quotas, (quota) => new Quota(quota)));
   }
+
+  getOrderFollowUp(orderId) {
+    const endPoint = `/me/order/${orderId}/followUp`;
+    return this.$http.get(endPoint).then(({ data }) => data);
```

Comment:
`.catch` not needed here? What if the API call fails?

Reply:
No @antleblanc , we used the then only to return directly the data without the angularjs extra info like header, status, etc ...
The catch must be implemented where the endpoint it used.